app/modules/entities/views/entities_list_adder.js

```diff
@@ -74,6 +74,12 @@ export default Marionette.CompositeView.extend({
       claims['wdt:P123'] = parentModel.get('claims.wdt:P123')
     }
 
+    if (parentModel.get('claims.wdt:P31')?.length === 1) {
```

Comment:
could we extract that in a named function to be more explicit about whats going on here ?

Reply:
attempted in 8beb8c0c